.evergreen.yml

```diff
@@ -122,6 +122,50 @@ tasks:
           permissions: public-read
           content_type: ${content_type|application/x-gzip}
 
+  - name: publish
+    # This should prevent this task from running in patch builds, where we
+    # don't want to publish packages.
+    patchable: false
+    stepback: false
+    depends_on:
+      - name: compile
+      - name: package
+    commands:
+      - func: "clone"
+      - func: "set up notary client credentials"
+      - command: shell.exec
+        params:
+          working_dir: mongocli
+          script: |
+            . ./notary_env.sh
+
+            set -o errexit
+            set -o verbose
+
+            CURATOR_RELEASE=${curator_release|"latest"}
+            curl -L -O http://boxes.10gen.com/build/curator/curator-dist-rhel70-$CURATOR_RELEASE.tar.gz
+            tar -zxvf curator-dist-rhel70-$CURATOR_RELEASE.tar.gz
+
+            # We get the raw version string (v1.2.3-45-gabcdef) from git
+            VERSION=$(git describe --abbrev=7 | cut -d "v" -f 2)
+
+            ./curator repo submit \
+              --service ${barque_url} \
+              --config .repo_config.yml \
+              --distro rhel70 \
+              --edition org \
+              --version $VERSION \
+              --arch x86_64 \
+              --packages https://s3.amazonaws.com/mongodb-mongocli-build/${project}/${revision}/dist/mongocli_next_${revision}_${version_id}_${created_at}_linux_x86_64.rpm
+
+            ./curator repo submit \
```

Comment:
I'd do what you can to try and get org/community to run in parallel.

Reply:
yes, I was just thinking of this seeing how long this takes, I may need to run other distros in parallel as well

Reply:
there's basically no local work, it's all busy-waiting, so finding some way to not serialize these tasks is good (and largely, all the work is parallelized on the backend.